app/controllers/interests_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
 
   def audit_click
     VacancyGetMoreInfoClick.new(vacancy).track
-    Auditor::Audit.new(vacancy, "vacancy.get_more_information", current_session_id).log
+    Auditor::Audit.new(vacancy, "vacancy.get_more_information", current_publisher_oid).log
```

Comment:
I'd vote for getting rid of all of these while you're here rather than updating them

Reply:
Agree they need to go sharpish, but I'd like to keep them around just a tiny bit longer – I want to replace them with analytics events as part of the events-to-Bigquery spike that's on the horizon.